client/aws/aws.go

```diff
@@ -23,36 +23,40 @@ func NewClient() Client {
 	}
 }
 
-type rawJSON struct {
+// Rawjson is the api message for AWS pricing .json file
+type Rawjson struct {
```

Comment:
AWSPricing?

Reply:
Move this to Line 57 and comment what's the expected type for Product/Term

Reply:
AWSPricing may be a little inappropiate, for the package name is AWS.
Also, pricing may be a littel narrow, since this struct not only contains pricing info, but also the instance info.
wdyt?